### PR TITLE
Dark Mode: Export Key

### PR DESCRIPTION
**Pull request by darkwing**
ExportModalLight
ExportModalDark

### CODE REVIEW

File: ui/components/app/modals/export-private-key-modal/index.scss [hunk 1]
```diff
@@ -43,17 +43,20 @@
...
-    color: var(--dusty-gray);
+    color: var(--color-text-muted);
   }
 
   &__password--warning {
     @include H7;
 
     border-radius: 8px;
-    background-color: #fff6f6;
+    background-color: var(--color-error-inverse);
```
Comment (georgewrmarshall) [inline ui/components/app/modals/export-private-key-modal/index.scss:59]: I think you want --color-error-muted here
```suggestion
    background-color: var(--color-error-muted);
```

File: ui/components/app/modals/export-private-key-modal/index.scss [hunk 2]
```diff
@@ -43,17 +43,20 @@
...
     font-weight: 500;
     color: var(--color-error-default);
```
Comment (georgewrmarshall) [inline ui/components/app/modals/export-private-key-modal/index.scss:58]: This should be updated to --color-text-default and we could add a border: 1px solid var(--color-error-default)

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]

**Comment by darkwing** [comment]
Updated!

**Review by georgewrmarshall — DISMISSED** [review]
LGTM!

## Updated Screenshots


Screen Shot 2022-03-23 at 10 24 52 AM
Screen Shot 2022-03-23 at 10 25 18 AM

**Review by georgewrmarshall — APPROVED** [review]

**Review by GuillaumeRx — APPROVED** [review]

**Pull request merged**